src/pages/AllDao.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<a-row>
+		<a-col :span="12" :offset="6">
+			<div v-if="error">
+				<a-result status="error" title="Submission Failed" :sub-title="error">
+				</a-result>
+			</div>
+			<div v-else>
+				<a-form
+					:label-col="{ span: 12 }"
+					:wrapper-col="{ span: 12 }"
+					:model="formState"
+					name="All Dao"
+					autocomplete="off"
+					@finish="onFinish"
+					@finishFailed="onFinishFailed"
+					@validate-messages="validateMessages"
+				>
+					<a-form-item
+						label="Select DAO"
+						name="dao_id"
+						:rules="[{ required: true, type: 'number' }]"
+					>
+						<a-input-number v-model:value="formState.dao_id" />
+					</a-form-item>
+
+					<a-form-item :wrapper-col="{ offset: 10, span: 20 }">
+						<a-button type="primary" html-type="submit">Select</a-button>
+					</a-form-item>
+				</a-form>
+			</div>
+		</a-col>
+	</a-row>
+</template>
+
+<script lang="ts">
+import { validateMessages } from "@/constants/constant";
+import { searchForAccount, searchForApplication } from "@/indexer";
+import WalletStore from "@/store/WalletStore";
+import { defineComponent, reactive } from "vue";
+import DaoStore from "../store/DaoID";
+
+export default defineComponent({
+	name: "AllDao",
+	data() {
+		return {
+			error: "",
+		};
+	},
+	methods: {
+		onFinish(values: any) {
```

Comment:
We should avoid use `any` type.

Reply:
yes I know but antd form doesn't have any specific type mentioned in their docs, they also use any, that's why using any